### ISSUE TITLE
fast: test (setup-ish) fails against AWS

### ISSUE THREAD

**Issue by dimaqq** [HENNGE/aiodynamo#21]
```
    async def test_query_with_limit(client: Client, fast_table: TableName):
        big = "x" * 20_000

>       await asyncio.gather(
            *(
                client.put_item(fast_table, {"h": "h", "r": str(i), "big": big})
                for i in range(100)
            )
        )

tests/integration/test_client.py:239:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src/aiodynamo/client.py:483: in put_item
    resp = await self.send_request(action="PutItem", payload=payload)
src/aiodynamo/client.py:650: in send_request
    return await self.http.post(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = HTTPX(client=<httpx.client.AsyncClient object at 0x7fd9914d37f0>)

    async def post(
        self, *, url: URL, body: bytes, headers: Optional[Headers] = None
    ) -> Dict[str, Any]:
        response = await self.client.post(str(url), data=body, headers=headers)
        if response.status_code >= 400:
>           raise exception_from_response(response.status_code, await response.aread())
E           aiodynamo.errors.ProvisionedThroughputExceeded: {'__type': 'com.amazonaws.dynamodb.v20120810#ProvisionedThroughputExceededException', 'message': 'The level of configured provisioned throughput for the table was exceeded. Consider increasing your provisioning level with the UpdateTable API.'}
```

I've hacked it to create tables with 1000/1000 rcu (?) and yet, test cannot insert 100 items concurrently.
😠

**Comment by ojii** [comment]
> I've hacked it to create tables with 1000/1000 rcu (?) and yet, test cannot insert 100 items concurrently.

is this on dynalite or real dynamodb?

**Comment by ojii** [comment]
`test_query_with_limit` requires ~2100 Write Capacity Units (WCU = (size(item) / 1KB) * num_items), because the items are intentionally "big" (~20kb)

**Comment by dimaqq** [comment]
real dynamo ofc, dynalite doesn't enforce limits :)

**Comment by ojii** [comment]
fixed in 20.3-rc4 https://github.com/HENNGE/aiodynamo/tree/20.3-rc4
